### PR TITLE
Replace Kantive-Eventing with the new Kyma-Eventing

### PR DESCRIPTION
**Pull request by radufa**
**Description**
Replace Kantive-Eventing with the new Kyma-Eventing
Changes proposed in this pull request:

- Don't provision Knative-Eventing
- Provision the new Kyma-Eventing
- Deprovision Kyma-Eventing

**Related issue(s)**

### CODE REVIEW

File: components/kyma-environment-broker/internal/cls/kyma_version_check_test.go [hunk 1]
```diff
@@ -24,3 +24,50 @@ func TestCheckKymaVersionPR(t *testing.T) {
...
+		summary string
+		version string
+		res     bool
+	}{
+		{"Check PR image", "PR-1234", true},
+		{"Check Master image", "master-abcd2", true},
+		{"Check Kyma image 1.21.1", "1.21.1", true},
+		{"Check Kyma image 1.21.0", "1.21.0", true},
+		{"Check Kyma image 1.20.0", "1.20.0", false},
+		{"Check Kyma image 1.20.0-rc4", "1.20.0-rc4", true},
```
Comment (piotrmiskiewicz) [inline components/kyma-environment-broker/internal/cls/kyma_version_check_test.go:63]: 1.20.0-rc4 is not 1.21

Comment (radufa) [reply components/kyma-environment-broker/internal/cls/kyma_version_check_test.go:63]: I put it as documentation, cause a non semantic-version string, but without errors, is mapped to "true" like that for "PR-1234" and "master-abcd2". This is an old behavior of the package.

**Review by piotrmiskiewicz — DISMISSED** [review]
lgtm

**Review by piotrmiskiewicz — APPROVED** [review]

**Pull request merged**